cookbook/doctrine/registration_form.rst

```diff
@@ -10,13 +11,13 @@ database. For example, you may want to create a registration form with some
 extra fields (like a "terms accepted" checkbox field) and embed the form
 that actually stores the account information.
 
-The simple User Model
----------------------
+The simple User Entity
```

Comment:
it's not your fault, but this should actually be: The Simple User Entity

And can you please add a placeholder for the old headline:

````
.. _the-simple-user-model:

The Simple User Entity
----------------------
````